omegaconf/basecontainer.py

```diff
@@ -225,9 +226,11 @@ def pretty(self, resolve: bool = False, sort_keys: bool = False) -> str:
         :param sort_keys: If True, will print dict keys in sorted order. default False.
         :return: A string containing the yaml representation.
         """
-        from omegaconf import OmegaConf
 
-        dumper = yaml.Dumper
+        return self._dump(resolve=resolve, sort_keys=sort_keys)
+
+    def _dump(self, stream=None, Dumper=DumperWrapper, **kwds) -> str:  # type: ignore
```

Comment:
1. I just meant you should call dump passing a Dumper, is this the function _dump adding anything?
2. I don't like DumperWrapper, maybe OmegaConfDumper? (same for the other Wrapper, by the way).

Reply:
1. Ah okay i thought you wanted a separate function.

2. Sure